src/vs/base/browser/ui/iconLabel/iconLabel.ts

```diff
@@ -103,10 +105,12 @@ export class IconLabel extends Disposable {
 
 	private readonly hoverDelegate: IHoverDelegate;
 	private readonly customHovers: Map<HTMLElement, IDisposable> = new Map();
+	private currentSupportIcons: boolean = false;
```

Comment:
I cannot wrap my head around this field, can we do this differently? It seems complex, and used for the description only? Could we get this differently?